infrastructure/shared/docker-compose-mt-keycloak.yml

```diff
@@ -147,7 +147,7 @@ services:
       - swagger-ui
 
   keycloak:
-    image: quay.io/keycloak/keycloak:23.0.7
+    image: quay.io/keycloak/keycloak:24.0.3
```

Comment:
This should match the version of client used in agent `23.0.7`

Reply:
its have the same erros em I go back to 23.0.7